Documentation/api.md

```diff
@@ -1052,6 +1052,7 @@ curl http://127.0.0.1:2379/v2/stats/self
 ### Store Statistics
 
 The store statistics include information about the operations that this node has handled.
+Note that v2 `store` is in-memory store. When a node stops, store statistics data will reset on restart.
```

Comment:
also we need to update https://github.com/coreos/etcd/blob/master/Documentation/metrics.md